### PR TITLE
Move OpenClinica from custom to apps.

### PR DESCRIPTION
**Pull request by kaapstorm**
This commit is to use the OpenClinica toggle to hide the report and export, as we do for OpenClinica settings.

Not sure how we decide about this kind of thing, so this PR is just to get feedback. Also not sure who to ping on this either, so I'll just try @snopoke @czue @dannyroberts and cc buddy @NoahCarnahan

### CODE REVIEW

File: corehq/apps/openclinica/reports.py [hunk 1]
```diff
@@ -122,7 +127,7 @@ def get_all_rows(self):
         CdiscOdmExportWriter will render this using the odm_export.xml template to combine subjects into a single
         ODM XML document.
         """
-        audit_log_id_ref = {'id': 0}  # To exclude audit logs, set `custom.openclinica.const.AUDIT_LOGS = False`
+        audit_log_id_ref = {'id': 0}  # To exclude audit logs, set `corehq.apps.openclinica.const.AUDIT_LOGS = False`
```
Comment (dimagimon) [inline corehq/apps/openclinica/reports.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/openclinica/reports.py [hunk 2]
```diff
@@ -98,7 +103,7 @@ def is_subject_selected(case):
 
     @property
     def rows(self):
-        audit_log_id_ref = {'id': 0}  # To exclude audit logs, set `custom.openclinica.const.AUDIT_LOGS = False`
+        audit_log_id_ref = {'id': 0}  # To exclude audit logs, set `corehq.apps.openclinica.const.AUDIT_LOGS = False`
```
Comment (dimagimon) [inline corehq/apps/openclinica/reports.py]: E501 line too long (117 > 115 characters)

**Comment by snopoke** [comment]
What's the reason for moving it?

**Comment by kaapstorm** [comment]
OpenClinica settings are hidden behind a flag, and creating an export document for OpenClinica is a custom report. So to use OpenClinica you have to enable the flag and also [include the domain in settings.py][1] for custom reports. That feels a bit silly, and I wondered whether it would be more consistent to use the flag only.

  [1]: https://github.com/dimagi/commcare-hq/blob/0b5d9a323801181929fbe05c26af18b4ab827037/settings.py#L1759-L1759

**Comment by snopoke** [comment]
That does make sense but I don't think it requires moving all the code.
Couldn't you just add the report to the main reports list and use the flag
to determine if it is visible?

**Comment by kaapstorm** [comment]
:+1: That sounds much less disruptive, and avoids having to create a new Django app.

**Pull request closed without merging**